src/api/Dockerfile

```diff
@@ -1,30 +1,75 @@
-# Node Alpine Docker Image
-FROM node:18.9-alpine3.16
+###################
+# 1. BUILD
+###################
 
-# Variables
+# NPM 8.19.1 Alpine linux image
+FROM node:18.15-alpine3.16 AS build
+
+# Build variables
 ARG GITHUB_SHA
 ARG SESSION_SECRET
 ARG GOV_NOTIFY_API_KEY
-ENV GITHUB_SHA $GITHUB_SHA
-ENV SESSION_SECRET $SESSION_SECRET
-ENV GOV_NOTIFY_API_KEY $GOV_NOTIFY_API_KEY
+ARG PORT
+ENV GITHUB_SHA=$GITHUB_SHA
```

Comment:
do we need `=`? I thought we can use a space between the variable and value

Reply:
Either is acceptable, I referred to official docker build guideline https://docs.docker.com/engine/reference/builder/.

Reply:
ok cool